usefulUtilities/voteApp/appResources/resources/config.js

```diff
@@ -0,0 +1,37 @@
+/* global module */
+
+var EVENT_DATE = "11_17_2018",
```

Comment:
It appears this data is duplicated at line 19 - how come?

Also, for purposes of putting this on GitHub, please replace specific data like this with generic data. In this case, perhaps consider something like "01_01_1970".